src/slotscheck/common.py

```diff
@@ -24,8 +24,15 @@ def cli(modulename: str, verbose: bool) -> None:
         stream=sys.stderr,
         level=logging.INFO if verbose else logging.ERROR,
     )
+
+    try:
+        module = importlib.import_module(modulename)
+    except ModuleNotFoundError:
+        print(f"ERROR: could not import module '{modulename}'")
+        return
```

Comment:
Nice! let's make sure we get an error code though on the command line. `exit(1)` should do it instead of `return`.